.github/workflows/bump-channel-version/action.yml

```diff
@@ -28,6 +28,10 @@ runs:
             echo "ELEMENTOR_CHANNEL_PACKAGE_VERSION=${ELEMENTOR_CHANNEL_PACKAGE_VERSION}"
             echo "NEW_CHANNEL_VERSION=${NEW_CHANNEL_VERSION}"
 
+            if [[ "${{ env.POSTFIX }}" == "-test" ]]; then
+              echo "PACKAGE_VERSION=${ELEMENTOR_CHANNEL_PACKAGE_VERSION}${{ env.POSTFIX }}" >> $GITHUB_ENV
```

Comment:
Why do we need to introduce a new env var (`PACKAGE_VERSION`)?

Reply:
As we talked about, `PACKAGE_VERSION` must be overridden in order to receive the following release name pattern
`3.16.0-cloud5-test` 
the build and release steps in the main action require the `PACKAGE_VERSION` var to name the release and the build file.
Previously this action was done by the publish -v2 action. but now, using the one-click action we need to imitate the naming action.